Lib/test/test_asyncio/test_staggered.py

```diff
@@ -121,6 +121,25 @@ async def coro(index):
         self.assertIsInstance(excs[0], ValueError)
         self.assertIsNone(excs[1])
 
+    def test_loop_argument(self):
```

Comment:
Ah, so this is just the original test restored. See my comment https://github.com/python/cpython/issues/124639#issuecomment-2381008468 for why I'm not yet approving this.

Reply:
Hm, I'm not so sure there ever were tests for staggered in 3.12. What I thought I saw was probably in a different branch. :-( Sorry.

Reply:
There was no tests for it and still has less in main than it should